scripts/test.sh

```diff
@@ -6,6 +6,11 @@ source ~/.nvm/nvm.sh
 if [[ "$(cut -d. -f1 <<< "$TRAVIS_NODE_VERSION")" -le 6 ]]; then
   nvm use 8
   yarn install --ignore-engines --ignore-scripts
+  # current versions of nock don't support node 6
+  cd packages/node
+  yarn add --dev --ignore-engines nock@10.x
+  yarn list --pattern nock
```

Comment:
Any reason why it's left in here?

Reply:
Nope - just missed taking it out. Thanks for catching.